bugbug/issue_features.py

```diff
@@ -74,9 +74,11 @@ def transform(self, issues):
 
             title = issue["title"]
             body = issue["body"]
-            for cleanup_function in self.cleanup_functions:
-                title = cleanup_function(title)
-                body = cleanup_function(body)
+
+            if issue.get("type") != "wcrt":
+                for cleanup_function in self.cleanup_functions:
+                    title = cleanup_function(title)
+                    body = cleanup_function(body)
```

Comment:
If you apply @marco-c's suggestion mentioned in https://github.com/mozilla/bugbug/pull/3790#discussion_r1388712411, then we will not need this workaround.

Reply:
Ok, I've added `items_gen` function to the model class and applied the cleanup for the text